examples/fluent-bit/filter-multiline/permissions.json

```diff
@@ -0,0 +1,13 @@
+{
+	"Version": "2012-10-17",
+	"Statement": [{
+		"Effect": "Allow",
+		"Action": [
+			"logs:CreateLogStream",
+			"logs:CreateLogGroup",
+			"logs:DescribeLogStreams",
+			"logs:PutLogEvents"
+		],
+		"Resource": "*"
+	}]
+}
```

Comment:
We didn't say anything about this policy. Do want want to say where and how to use this?

Reply:
It is for sending to cloudwatch and it is introduced in other examples so I assume customers use this example just focus on how to use the filter. Do we also need to add details for other parts?